src/apiClient.ts

```diff
@@ -136,11 +117,7 @@ export default function APIClient(
         this.processQueuedEvents();
 
         if (event && options.shouldUploadEvent) {
-            if (this.shouldEnableBatching()) {
-                this.queueEventForBatchUpload(event);
-            } else {
-                this.sendSingleEventToServer(event);
-            }
+            this.queueEventForBatchUpload(event);
```

Comment:
Not sure if this is out of scope, but since both this block and the one below check to see if `event` is empty, can we make the following refactor:

```
if (isEmpty(event)) {
    return;
}

if (options.shouldUploadEvent) {
    this.queueEventForBatchUpload(event);
}

if (event.EventName !== Types.MessageType.AppStateTransition) {
    // kit blocking code
}
```

Reply:
Good call